### PR TITLE
Hunting queries windows event batch2

### PR DESCRIPTION
**Pull request by yaelrbergman**
Required items, please complete
   
   Change(s):
Add Windows Event Data to the following Hunting Queries:
Hunting Queries/MultipleDataSources/UnicodeObfuscationInCommandLine.yaml
Hunting Queries/SecurityEvent/cscript_summary.yaml
Hunting Queries/SecurityEvent/ExchangePowerShellSnapin.yaml
Hunting Queries/SecurityEvent/new_processes.yaml
Hunting Queries/SecurityEvent/PowerCatDownload.yaml
Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml
Hunting Queries/SecurityEvent/SuspectedLSASSDump.yaml

Reason for Change(s):
part of adding Windows Event data source project

Version Updated:
No
Testing Completed:
Yes
Checked that the validations are passing and have addressed any issues that are present:
Yes

### CODE REVIEW

File: Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml [hunk 1]
```diff
@@ -16,11 +19,22 @@ query: |
...
-  let processEvents=SecurityEvent
+  let processEvents=(union isfuzzy=true  (SecurityEvent
   | where TimeGenerated between(ago(lookback)..endtime)
   | where EventID==4688
   | where NewProcessName has_any ("powershell.exe","pwsh.exe")
-  | project TimeGenerated, Computer, Account, NewProcessName, FileName=tostring(split(NewProcessName, '\\')[-1]), ProcessCommandLine = CommandLine, ParentProcessName;
+  | project TimeGenerated, Computer, Account, NewProcessName, FileName=tostring(split(NewProcessName, '\\')[-1]), ProcessCommandLine = CommandLine, ParentProcessName),
+  (WindowsEvent
+  | where TimeGenerated between(ago(lookback)..endtime)
+  | where EventID==4688 and EventData has_any ("powershell.exe","pwsh.exe")
```
Comment (oshezaf) [inline Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml:29]: ass and has "-encodedCommand"

**Review by oshezaf — CHANGES_REQUESTED** [review]

**Comment by shainw** [comment]
@oshezaf - looks like this can be approved?

**Comment by vmanojreddy** [comment]
@oshezaf  Please check and approve the changes, thanks.

**Comment by vmanojreddy** [comment]
@oshezaf Please check and approve the changes, thanks.

**Comment by v-spadarthi** [comment]
@oshezaf : Please check and approve the changes, thanks.

**Comment by v-mchatla** [comment]
Hi @oshezaf
Can we close this and reopen whenever required as you have labelled it as differed.
Thanks

**Comment by oshezaf** [comment]
Done

**Pull request closed without merging**